zerver/lib/test_fixtures.py

```diff
@@ -413,7 +413,7 @@ def reset_zulip_test_database() -> None:
 
     destroy_test_databases()
     # Pointing default database to test database template, so we can instantly clone it.
-    settings.DATABASES["default"]["NAME"] = settings.BACKEND_DATABASE_TEMPLATE
+    settings.DATABASES["default"]["NAME"] = "zulip_test_template"
```

Comment:
Can one of `text_fixtures.py` and `test_runner.py` import this constant from the other?

We generally prefer that sort of model over having the same string in two places, if the intent is they should always be the same.

Reply:
Fixed.